setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='pyicloud',
-    version='0.7.3',
+    version='0.7.4',
```

Comment:
Sounds good to me!  For us to follow semantic versioning as well as we can, you'll probably want a minor version bump (since this adds new features) rather than a patch bump (which is only really for bug fixes).